Holyhead-Harpies/cli-sprint2-jw
# Add product model and controller with tests

## Status
IN DEVELOPMENT
Issue: #4 

## Migrations
NO

## Description
Product Add Model and Controller



## Todos



## Steps to Test or Reproduce

1.
```
git fetch origin add_product_bl
git checkout add_product_bl
ruby db/create_tables.rb
```

2.
```
rspec spec/product_controller_spec.rb
rspec spec/product_model_spec.rb
``` 

All tests should pass.


## Impacted Areas in Application
List general components of the application that this PR will affect:

*Product controller/model/test files/Product table on database

## Impacted Areas in Application
List general components of the application that this PR will affect:

*
